format/FlightSql.proto

```diff
@@ -1475,6 +1577,35 @@ message CommandStatementQuery {
 
   // The SQL syntax.
   string query = 1;
+  // Include the query as part of this transaction (if unset, the query is auto-committed).
+  bytes transaction_id = 2;
```

Comment:
These `transaction_id` fields should be explicitly marked `optional`

Reply:
technically, by definition in proto3 all fields are optional and it's unnecessary to explicitly mark any as such.